test/gscraper_web/plugs/skip_after_auth_plug_test.exs

```diff
@@ -0,0 +1,29 @@
+defmodule GscraperWeb.Plugs.SkipAfterAuthPlugTest do
+  use GscraperWeb.ConnCase, async: true
+
+  alias GscraperWeb.Plugs.SkipAfterAuthPlug
+
+  describe "init/1" do
+    test "returns given options" do
+      assert SkipAfterAuthPlug.init([]) == []
+    end
+  end
+
+  test "renders to the dashboard page given the user already logged in", %{conn: conn} do
```

Comment:
I think this test and the below one wrapped in `call/2` describe block.

Reply:
that's correct. I fixed it in 1c2972e 🙈